src/diffusers/callbacks.py

```diff
@@ -0,0 +1,156 @@
+from typing import Any, Dict, List
+
+from .configuration_utils import ConfigMixin, register_to_config
+from .utils import CONFIG_NAME
+
+
+class PipelineCallback(ConfigMixin):
```

Comment:
How does this class benefit from inheriting from `ConfigMixin`? It's just two init arguments which likely are not going to be changed. 

If it's not an absolute necessity, I would prefer to just have it as `class PipelineCallback:`. This way the users won't have to worry about learning a new concept while reading the code.

Reply:
I had them as instance variables but @yiyixuxu suggested this so I thought it was the way to do it in the diffusers philosophy. Just take into account that in the future there could be more variables if needed, for example If we add a `on_step_begin_callback`.